.github/pull_request_template.md

```diff
@@ -15,9 +15,8 @@ To get an auto-generated PR description you can put "copilot:summary" or "copilo
 ### Checklist
 * [ ] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
 * [ ] I've included a screenshot or gif (if applicable)
-* [ ] I have tested https://demo.rerun.io/pr/{{ pr-number }} (if applicable)
+* [ ] I have tested [demo.rerun.io](https://demo.rerun.io/pr/{{ pr.number }}) (if applicable)
 
-<!-- This line will get updated when the PR build summary job finishes. -->
-PR Build Summary: {{ pr-build-summary }}
-
-<!-- This comment will be replaced by a link to the documentation preview -->
+- [PR Build Summary](https://build.rerun.io/pr/{{ pr.number }})
+- [Docs preview](https://rerun.io/preview/{{ "pr:%s"|format(pr.branch)|encode_uri_component }}/docs)
```

Comment:
The links to the previews are a bit weird. The end result looks like
```
https://rerun.io/preview/pr:jan/pull-request/examples
```
But with `pr:jan/pull-request` url-encoded.

Motivations for this:
- It's treated as a single path segment this way
- The github `.zip` archive link is different for branches, so the `pr:` prefix means we can differentiate between regular commit previews and branch previews, and choose the correct path.

Commit previews continue to work as before.